plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/file/HudiBaseFile.java

```diff
@@ -0,0 +1,75 @@
+package io.trino.plugin.hudi.file;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.hudi.common.model.HoodieBaseFile;
+
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class HudiBaseFile implements HudiFile {
```

Comment:
If this is the only implementation of HudiFile then do we need the interface? Maybe HudiBaseFile can  simply be a record.

Reply:
Yes